lib/hydra/derivatives/processors/jpeg2k_image.rb

```diff
@@ -73,20 +73,18 @@ def layer_rates(layer_count, compression_numerator)
         rates.map(&:to_s).join(',')
       end
 
-      protected
```

Comment:
Here's an older explanation of why it won't work to have a protected class method and then try to call it from within an instance method.  The only receive that is allowed is `self`;  `self.class` won't work.
http://stackoverflow.com/questions/11461262/what-is-the-value-of-self-in-a-rails-model-and-why-arent-obvious-instance-metho/11461333#11461333